README.md

```diff
@@ -2,25 +2,31 @@
 
 Ansible modules for interacting with the Foreman API and various plugin APIs such as Katello.
 
-## Goals
-
-The intent of this repository is to be a place that community members can develop or contribute modules. The goals of this repository are:
-
-  * centralized location for community modules
-  * a single repository to clone for interacting with Foreman & plugins
-  * source for the official Ansible collection
-
 ## Documentation
 
 A list of all modules and their documentation can be found at [theforeman.org/plugins/foreman-ansible-modules](https://theforeman.org/plugins/foreman-ansible-modules/).
 
-## Supported Foreman and plugins versions
+Documentation how to [write](docs/developing.md), [test](docs/testing.md) and [debug](docs/debugging.md) modules is available in the [`docs`](docs/) folder.
```

Comment:
Are those links working when viewed from Galaxy?
And i guess, i know the answer... Probably not.

Reply:
probably not, no.

Reply:
definitely not, just tested with the (old) `tests/README.md` link we had, and it points to https://galaxy.ansible.com/tests/README.md :see_no_evil: